diff -urN mate-desktop-1.8.0.orig/desktop-docs/fdl/Makefile.am mate-desktop-1.8.0/desktop-docs/fdl/Makefile.am
--- mate-desktop-1.8.0.orig/desktop-docs/fdl/Makefile.am	2014-03-04 23:51:34.610268291 +0100
+++ mate-desktop-1.8.0/desktop-docs/fdl/Makefile.am	2014-03-04 23:52:02.000267487 +0100
@@ -1,6 +1,6 @@
 @YELP_HELP_RULES@
 
-HELP_ID = fdl
+HELP_ID = mate-fdl
 HELP_FILES = index.docbook
 HELP_LINGUAS = ar ca el es eu fr hu ko oc pl sv uk vi de
 
diff -urN mate-desktop-1.8.0.orig/desktop-docs/gpl/Makefile.am mate-desktop-1.8.0/desktop-docs/gpl/Makefile.am
--- mate-desktop-1.8.0.orig/desktop-docs/gpl/Makefile.am	2014-03-04 23:51:34.610268291 +0100
+++ mate-desktop-1.8.0/desktop-docs/gpl/Makefile.am	2014-03-04 23:52:06.646933945 +0100
@@ -1,6 +1,6 @@
 @YELP_HELP_RULES@
 
-HELP_ID = gpl
+HELP_ID = mate-gpl
 HELP_FILES = index.docbook
 HELP_LINGUAS = ar ca cs el es eu fi fr hu ko nds oc pa uk vi zh_CN sv de
 
diff -urN mate-desktop-1.8.0.orig/desktop-docs/lgpl/Makefile.am mate-desktop-1.8.0/desktop-docs/lgpl/Makefile.am
--- mate-desktop-1.8.0.orig/desktop-docs/lgpl/Makefile.am	2014-03-04 23:51:34.613601629 +0100
+++ mate-desktop-1.8.0/desktop-docs/lgpl/Makefile.am	2014-03-04 23:52:10.910267016 +0100
@@ -1,6 +1,6 @@
 @YELP_HELP_RULES@
 
-HELP_ID = lgpl
+HELP_ID = mate-lgpl
 HELP_FILES = index.docbook
 HELP_LINGUAS = ar de el es eu fi fr hu ko oc pa sv uk vi zh_CN
 
